README.adoc

```diff
@@ -22,7 +22,7 @@ suite runs cleanly and should remain clean.
 
 You can format the code using `cargo fmt`.  CI checks that code is correctly formatted.
 
-https://github.com/rust-lang/rust-clippy[Clippy] is used to check for common code issues.  (We disable the style checks.)  You can run it with `cargo clippy`.  CI will run clippy as well.
+https://github.com/rust-lang/rust-clippy[Clippy] is used to check for common code issues.  (We disable the style checks.)  You can run it with `cargo clippy --all-targets -- --deny warnings --allow clippy::style`.  CI will run clippy as well.
```

Comment:
Totally.  I think I've seen mechanisms for running GitHub Actions jobs locally, which has some nice properties (what you're doing matches CI by construction) but also some annoying ones (what environment does that run in? is it the same as CI? does that mean you need a whole VM?).

Reply:
I think it would help if the Rust tools were more reliably configurable in the repo.  For example, you can put the `--allow clippy::style` into the Rust code, and if you could do that with everything, you could just tell people to run `cargo clippy`.  It doesn't work here partly because of the bug mentioned in the workflow file, and also because of needing to include stuff like `--all-targets`.  I assume I'm not the only one who forgets to tack that on all the time.